File: day12_inheritance.py
class Grade(Student):
    def __init__(self,firstName, lastName, phone, score):
        super().__init__(firstName,lastName, phone)
        self.score = score
    def calculate(self):
        if self.score < 40:
            return "D"
        elif self.score >= 40 and self.score < 60:
            return "B"
        elif self.score >= 60 and self.score < 75:
            return "A"
        elif self.score >= 75 and self.score < 90:
            return "E"
        elif self.score >= 90 and self.score <= 100:
            return "O"
/

File: day12_inheritance.cpp
class Grade :  public Student{
    private:
         int score; 
    public:
    Grade(string fname,string lname,int p,int score):Student(fname,lname,p) {this->score=score;}
    char calculate(){
        char Grade;
        if( score < 40 )
            
        {Grade='D';}
            else if(score>=40&&score<60)
            {Grade='B';}
            else if(score>=60&&score<75)
            {Grade='A';}
            else if (score>=75&&score<90)
            {Grade='E';}
            else if(score>=90&&score<=100)
            {Grade='O';}
                return Grade;
    }
};
